test/e2e/step-definitions/waitingroom-steps.ts

```diff
@@ -18,7 +18,7 @@ Before({ tags: '@catc' }, () => {
   this.testCategory = 'c';
 });
 
-Before({ tags: '@catc1' }, () => {
+Before({ tags: '@catc1e' }, () => {
```

Comment:
We need this and we already have this tag

Reply:
done